### ISSUE TITLE
Does please work on windows?

### ISSUE THREAD

**Issue by santagada** [thought-machine/please#227]
Is anyone working on it?

**Comment by peterebden** [comment]
tl;dr: Probably not, and nope :(

> Does please work on windows?

Seems unlikely, the build rules use Unix coreutils for quite a few things. It's possible that it might work under their new bash/ubuntu on Windows setup (or possibly cygwin) but very unlikely under normal win32.

I suspect there would be some work to get the cffi / pypy embedding working too, although it might not be difficult.

> Is anyone working on it?

Nope, none of us have a Windows development setup, and we only have Linux and Mac build agents, so it's pretty hard for us to support.

I'd be happy to accept targeted pull requests that improve the compatibility situation, as long as it doesn't require a wholesale rewrite of it the whole thing.

**Comment by santagada** [comment]
Thanks, just explaining what needs to be changed to support windows helps a lot :)
